stac_api/runtime/handler.py

```diff
@@ -4,12 +4,15 @@
 
 from mangum import Mangum
 from src.app import app
+from src.config import ApiSettings
 from src.monitoring import logger, metrics, tracer
 
+settings = ApiSettings()
+
 logging.getLogger("mangum.lifespan").setLevel(logging.ERROR)
 logging.getLogger("mangum.http").setLevel(logging.ERROR)
 
-handler = Mangum(app, lifespan="auto")
+handler = Mangum(app, lifespan="auto", api_gateway_base_path=settings.root_path)
```

Comment:
instead of using `settings` I think you can use `app.root_path` 

ref https://github.com/developmentseed/eoapi-cdk/pull/82/files#diff-62c863ae9edf662fc81920ac522a18c5650fb0a927fb36b0077e079c4999c88dR9

Reply:
@smohiudd @ividito: after making 👆 suggested change, I was also able to remove the additional fast api additional servers configuration (sharing because was a red flag in reviews yesterday)